src/features/content-explorer/content-explorer/__tests__/ContentExplorerActionButtons.test.js

```diff
@@ -324,16 +324,61 @@ describe('features/content-explorer/content-explorer/ContentExplorerActionButton
     });
 
     describe('renderStatus', () => {
-        test('should show status message for multi select', () => {
-            const item = { id: '0', name: 'item1' };
-            const selectedItems = { '0': item };
+        let item;
+        let selectedItems;
+
+        beforeEach(() => {
+            item = { id: '0', name: 'item1' };
+            selectedItems = { '0': item };
+        });
+
+        test('should show default status message for multi select', () => {
             const wrapper = renderComponent({
                 contentExplorerMode: ContentExplorerModes.MULTI_SELECT,
                 selectedItems,
             });
+            const statusMessageId = wrapper
+                .find('.status-message')
+                .find('FormattedMessage')
+                .prop('id');
 
-            expect(wrapper.find('.status-message').length).toEqual(1);
+            expect(statusMessageId).toEqual('boxui.contentExplorer.numSelected');
```

Comment:
nit: could we use `toBe` for this and below? the repo has historically preferred `toBe` for primitive data types